passtrain.py

```diff
@@ -18,9 +18,14 @@ def attempt_submission(true_passphrase):
    parser = argparse.ArgumentParser()
    parser.add_argument("-n", "--num_consec", dest="num_consec", type=int, default="10", 
                        help="number of consecutive successes before exit")
+   parser.add_argument("-i", "--invisible", action="store_true",
+                       help="do not show any input on screen")
    args = parser.parse_args()
 
-   passphrase = input("Enter passphrase to remember: ")
+   if not args.invisible:
+      passphrase = input("Enter passphrase to remember: ")
+   else:
+      passphrase = getpass.getpass("Enter passphrase to remember: ")
```

Comment:
Using `getpass` here is definitely the right call - want to just make this the default?